android/app/src/main/res/values/dimens.xml

```diff
@@ -15,5 +15,7 @@
     <dimen name="xxxxxl">112dp</dimen>
     <dimen name="xxxxxxl">128dp</dimen>
     <dimen name="xxxxxxxl">144dp</dimen>
+    <dimen name="xxxxxxxxl">160dp</dimen>
+    <dimen name="xxxxxxxxxl">176dp</dimen>
```

Comment:
You probably shouldn't need to make dimen's this high.

Reply:
then what can i do to make a box this big?